index.html

```diff
@@ -3514,6 +3514,58 @@ <h3>Reserved Extension Points</h3>
         </p>
 
       </section>
+
+      <section class="normative">
+        <h3>Ecosystem Compatibility</h3>
+
+        <p>
+There are a number of digital credential formats that do not natively use the
+data model provided in this document, but are aligned with a number of concepts
+in this specification. At the time of publication, examples of these digital
+credential formats include pure JSON Web Tokens (JWTs), CBOR Web Tokens (CWTs),
+ISO-18013-5 (mDLs), and Authentic Chained Data Containers (ACDCs).
+        </p>
+
+        <p>
+If conceptually aligned digital credential formats can be transformed to the
+data model according to the rules provided in this section, they are considered
```

Comment:
```suggestion
`application/vc+ld+json` data model according to the rules provided in this section, they are considered
```

Reply:
+1 to @TallTed's suggestion here -- it's important to note that the transformation is to the base format.

Reply:
+1 to this.

Reply:
The reason I didn't put a specific media type here is because we have /two/ media types defined by this specification. The initial PR here wrote out both media types everywhere, and it was difficult to read. I can do that again if folks want, but it will be... difficult to read. :) 

Are the three of you sure you want to do this? We will have to mention `application/vc+ld+json` and `application/vp+ld+json` everywhere. Or, we can say `conformant document`, which includes both media types (and is why I tried used that language throughout).

Given that, what do we want to do here?

Reply:
any chance we can say `vc+ld+json` also works to sending JSON-LD with an `@context` and `type` ?.... seems we are wasting a media type on verifiable presentations... regardless, I prefer explicit over implicit here.

Reply:
It's fine with me if we're saying "conformant document" to avoid mentioning both media types, I just want to avoid saying "transforming to the data model" which is the wrong thing to say, IMO.

Reply:
I have made an attempt at re-writing the language to be more consistent and use "conforming document" in commit 08ad96a0b98540c6b01257d10e1b3faf22c07fd9 and then be specific about what a "conforming document" is in bd9aab2a283bd45789984fe2fc528c696ebe2aac.